src/client/pages/RetroCertsCertificationPage/index.js

```diff
@@ -30,9 +32,24 @@ function RetroCertsCertificationPage(props) {
     return <Redirect to={routes.retroCertsWeeksToCertify} />;
   }
 
-  const handleSubmit = (event) => {
-    // TODO: form validation, post form.
-  };
+  function handleSubmit() {
+    fetch(AUTH_STRINGS.apiPath.save, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        formData: mockedFormData,
+        authToken: sessionStorage.getItem(AUTH_STRINGS.authToken),
+      }),
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        setUserData(data);
+        history.push(routes.retroCertsConfirmation);
+      })
+      .catch((error) => console.error(error));
```

Comment:
We need to handle errors, right? Like If the auth token is incorrect. Maybe add a TODO?

Reply:
Yep! There are a few auth / error handling errors that I need to update. I'll do that as part of a separate PR.